docs/contributing/contributing-icons.md

```diff
@@ -0,0 +1,107 @@
+function GetIcons({ iconName, ...restProps }) {
+  switch (iconName) {
+    case 'android':
+      return <FaAndroid {...restProps} />
+    // ...
+
+    // spotify icon
+    // if `iconName` is "spotify", then it will return this icon
+    case 'spotify':
+      return <FaSpotify {...restProps} />
+    // ...
+  }
+}
+```
+
+After that open `links.json` file and add a new property to the `validIcons` object. The property name must be same as the icon name (in this case, `spotify`).
+
+```json
+{
+   "validIcons": {
+      "android": "#3DDC84",
+      "spotify": "#1DB954"
+   }
+}
+```
+
+Then open `icons.md` file and add the new icon name, usage and preview image to the icon table:
+
+```markdown
+# Icons
+
+<!-- ... -->
+
+|   Socials   |     Usage     | Icons                                                                                                                                                        |    Socials     |      Usage       | Icons                                                                                                                                   |
+| :---------: | :-----------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------ | :------------: | :--------------: | --------------------------------------------------------------------------------------------------------------------------------------- |
+|   Android   |   `android`   | <img src="https://user-images.githubusercontent.com/65664185/138502465-89cfadf2-6b54-4f3d-ac44-ceacdd4824ba.png" width=65% height=30%>                       |     Apple      |     `apple`      | <img src="https://user-images.githubusercontent.com/65664185/138502540-8e9b80bf-deae-4566-a41a-c63623e83c21.png" width=100% height=30%> |
+|    Book     |    `book`     | <img src="https://user-images.githubusercontent.com/76985777/145391108-f8c08f8e-679f-45a3-ad58-83ef60aa28fe.png" width=65% height=30%>                       |   Codeforces   |   `codeforces`   | <img src="https://user-images.githubusercontent.com/91655303/148160942-870fdbb4-a57c-4861-afaa-241835390645.png" width=100% height=30%> |
+
+<!-- ... -->
+
+|   YouTube   |   `youtube`   | <img src="https://user-images.githubusercontent.com/65664185/138503305-ff60cf54-6b0b-4e18-9446-b4f6869b9511.png" width=65% height=30%>                       |    Spotify     |    `spotify`     | <img src="<icon_preview_url_here>" width="100%" height="30%">                                                                           |
+```
+
+And that's it! You can now commit, push, and [submit a PR](htttps://github.com/EddieHubCommunity/LinkFree/compare), and the maintainers will check your PR. 🙂
+
+## Committing and pushing the changes
+
+After you've finished making changes, you need to commit and push those first. 
+If you've cloned the forked repo locally and working on it, run these commands:
+
+```
+git add <files_you_have_changed>
```

Comment:
specify the exact git add command here so that noone is left guessing what to write for the files_you_have_changed